spec/helpers/service_helper/textual_summary_spec.rb

```diff
@@ -0,0 +1,30 @@
+describe ServiceHelper::TextualSummary do
+  context ".textual_orchestration_stack" do
```

Comment:
@skateman - I have changed it - but I do not really agree - context is an alias of describe - and usually - describe wraps the user story, and context the individual scenario. We, as a team  -can decide on a set of guidelines for how we mark our specs.